src/axom/core/CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ set(core_sources
 #------------------------------------------------------------------------------
 
 set( core_depends fmt )
+blt_list_append( TO core_depends ELEMENTS camp IF CAMP_FOUND )
```

Comment:
Do we use camp w/o umpire/raja? Not that it super matters...

Reply:
Not as far as I know. It looks like raja's import was not bringing in the camp include (and I suspect that Umpire's import was). Not sure if this has been fixed between `raja@2022.03` (our current version) and `raja@develop`